lib/solidity/bytes.js

```diff
@@ -2,37 +2,33 @@ var f = require('./formatters');
 var SolidityType = require('./type');
 
 /**
- * SolidityTypeBytes is a prootype that represents bytes type
- * It matches:
- * bytes
- * bytes[]
- * bytes[4]
- * bytes[][]
- * bytes[3][]
- * bytes[][6][], ...
- * bytes32
- * bytes64[]
- * bytes8[4]
- * bytes256[][]
- * bytes[3][]
- * bytes64[][6][], ...
+ * SolidityTypeBytes is a prototype that represents bytes type
+ * Given the correct size at initialization, it matches:
+ *   bytes32
+ *   bytes8[4]
+ *   bytes256[][]
```

Comment:
This really has to be refactored in the future. The whole point of the ABI is that you do not have to care about whether you are encoding an array of `bytes8` or of `bytes16[]`.

Plus, there is no type `bytes256`.